ui/admin/app/styles/app.scss

```diff
@@ -509,6 +510,7 @@
   height: var(--secret-editor-height);
   margin-bottom: sizing.rems(l);
 
+  /* stylelint-disable selector-class-pattern */
   .secret-editor-json {
     .CodeMirror {
```

Comment:
Is there a "next line" way to disable the rule?  That way developers can still add to the outer selector and be subject to linting.

Reply:
Alternatively maybe it's better to actually just add a custom regex for [this rule](https://stylelint.io/user-guide/rules/selector-class-pattern/) since it might come up more often and just add `CodeMirror` as an exception since it's not our class. 

Stylelint provides their regex for kebab case and we could just add to it for `CodeMirror` as well

Reply:
Indeed, I like @ZedLi idea!